src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -13,199 +13,385 @@ public enum SyntaxKind : ushort
         GetKeyword = 8417,
+        /// <summary>Represents <see langword="set"/>.</summary>
         SetKeyword = 8418,
+        /// <summary>Represents <see langword="add"/>.</summary>
         AddKeyword = 8419,
+        /// <summary>Represents <see langword="remove"/>.</summary>
         RemoveKeyword = 8420,
+        /// <summary>Represents <see langword="where"/>.</summary>
         WhereKeyword = 8421,
+        /// <summary>Represents <see langword="from"/>.</summary>
         FromKeyword = 8422,
+        /// <summary>Represents <see langword="group"/>.</summary>
         GroupKeyword = 8423,
+        /// <summary>Represents <see langword="join"/>.</summary>
         JoinKeyword = 8424,
+        /// <summary>Represents <see langword="into"/>.</summary>
         IntoKeyword = 8425,
+        /// <summary>Represents <see langword="let"/>.</summary>
         LetKeyword = 8426,
+        /// <summary>Represents <see langword="by"/>.</summary>
         ByKeyword = 8427,
+        /// <summary>Represents <see langword="select"/>.</summary>
         SelectKeyword = 8428,
+        /// <summary>Represents <see langword="orderby"/>.</summary>
         OrderByKeyword = 8429,
+        /// <summary>Represents <see langword="on"/>.</summary>
         OnKeyword = 8430,
+        /// <summary>Represents <see langword="equals"/>.</summary>
         EqualsKeyword = 8431,
+        /// <summary>Represents <see langword="ascending"/>.</summary>
         AscendingKeyword = 8432,
+        /// <summary>Represents <see langword="descending"/>.</summary>
         DescendingKeyword = 8433,
+        /// <summary>Represents <see langword="nameof"/>.</summary>
         NameOfKeyword = 8434,
+        /// <summary>Represents <see langword="async"/>.</summary>
         AsyncKeyword = 8435,
+        /// <summary>Represents <see langword="await"/>.</summary>
         AwaitKeyword = 8436,
+        /// <summary>Represents <see langword="when"/>.</summary>
         WhenKeyword = 8437,
+        /// <summary>Represents <see langword="or"/>.</summary>
         OrKeyword = 8438,
+        /// <summary>Represents <see langword="and"/>.</summary>
         AndKeyword = 8439,
+        /// <summary>Represents <see langword="not"/>.</summary>
         NotKeyword = 8440,
+        /// <summary>Represents <see langword="data"/>.</summary>
         DataKeyword = 8441,
+        /// <summary>Represents <see langword="with"/>.</summary>
         WithKeyword = 8442,
+        /// <summary>Represents <see langword="init"/>.</summary>
         InitKeyword = 8443,
+        /// <summary>Represents <see langword="record"/>.</summary>
         RecordKeyword = 8444,
+        /// <summary>Represents <see langword="managed"/>.</summary>
         ManagedKeyword = 8445,
+        /// <summary>Represents <see langword="unmanaged"/>.</summary>
         UnmanagedKeyword = 8446,
 
         /// when adding a contextual keyword following functions must be adapted:
```

Comment:
This line generates an error. It existed before my changes, not sure what has changed that made the error just appeared now?

Reply:
You added a second documentation comment for the element. Only one is allowed.

```suggestion
        // when adding a contextual keyword following functions must be adapted:
```